django/db/migrations/operations/models.py

```diff
@@ -147,6 +150,8 @@ def reduce(self, operation, in_between, app_label=None):
                     managers=self.managers,
                 ),
             ]
+        elif isinstance(operation, CreateModel):
+            return True
```

Comment:
Yikes, sorry for that. Github swallowed my update to my comment. I'm trying to remember the example I had yesterday. IIRC something along those lines. Don't recall the details, unfortunately :cry: 

When I have this migration:
```python
operations = [
    migrations.CreateModel('Person', [("id", models.AutoField(primary_key=True, ...)), ("last_name", models.CharField(max_length=255))]),
    migrations.CreateModel('Book', [("id", models.AutoField(primary_key=True, ...)), ('author', models.ForeignKey('test_app.Person', models.CASCADE, to_field='last_name'))]),
    migrations.RemoveField('person', 'first_name'),
    migrations.AlterField('book', 'author', models.ForeignKey('test_app.Person', models.CASCADE)),
]
```